spec/models/user_spec.rb

```diff
@@ -11,4 +11,57 @@
     it { should_not allow_value('name@').for(:email) }
     it { should_not allow_value('@email.com').for(:email) }
   end
+
+  describe '.send_payment_client' do
+    it  'should post to API when client is client_person' do
```

Comment:
esse método atua numa instância `user`, então se não me engano o símbolo é `#` (seria `.` se ele atuasse na classe `User`).

As expectativas dos testes ficariam mais claras se fossem tipo assim:
```suggestion
describe '#send_payment_client' do
  context 'should make a POST to create user at Payments app' do
    it 'when the client is a natural person (with CPF)' do
    end
    it 'when the client is a business (with CNPJ)' do
    end
  end
end
```

Reply:
Alterações incorporadas no teste.